clients/go/main.go

```diff
@@ -58,18 +63,26 @@ func main() {
 
 		log.Printf("Upload offset is: %d", offset)
 
+		uploadCompleteHeader := "?0"
```

Comment:
This client should always set Upload-Complete: ?1 because it tries to transmit the entire remaining data in the request. It means that if the data was received successfully the server should consider the upload complete. Upload-Complete: ?0 is only needed if the client wants to split the upload into multiple requests.

Reply:
Got it. Thanks. Have done all changes and committed.


p.s, This is my First Open source contribution. 😄